TradeItIosTicketSDK2/TradeItAlertProvider.swift

```diff
@@ -34,28 +34,41 @@ class TradeItAlertProvider {
 
     static func provideSecurityQuestionAlertWith(alertTitle alertTitle: String,
                                                             alertMessage: String,
+                                                            multipleOptions: [String],
                                                             alertActionTitle: String,
                                                             onAnswerSecurityQuestion: (withAnswer: String) -> Void,
                                                             onCancelSecurityQuestion: () -> Void) -> UIAlertController {
         let alertController = UIAlertController(title: alertTitle,
                                                 message: alertMessage,
                                                 preferredStyle: UIAlertControllerStyle.Alert)
 
-        alertController.addTextFieldWithConfigurationHandler(nil)
-
-        let submitAction = UIAlertAction(title: "Submit", style: .Default, handler: { (action) -> Void in
-            let textField = alertController.textFields![0] as UITextField
-            onAnswerSecurityQuestion(withAnswer: textField.text!)
-        })
-
         let cancelAction = UIAlertAction(
             title: "Cancel",
             style: UIAlertActionStyle.Default) { (result : UIAlertAction) -> Void in
                 onCancelSecurityQuestion()
+        }
+        
+        alertController.addAction(cancelAction)
+        
+        if multipleOptions.count > 0 {
+            for option in multipleOptions {
+                let optionAction = UIAlertAction(title: option, style: .Default, handler: { (action) -> Void in
```

Comment:
This is great as is, but just a small suggestion: you don't always need to provide the argument types or return types in these closures so `{ action in` works just fine and is still type checked because the `UIAlertAction` has already defined what type signature the handler should be.